test/test_other.py

```diff
@@ -9650,15 +9650,18 @@ def test_closure_full_js_library(self, args):
     for sym in glsyms:
       self.assertContained('.' + sym, js)
 
+  @crossplatform  # So tests will run on non-Linux platforms, where FROZEN_CACHE isn't set.
   @also_with_wasm64
   def test_closure_webgpu(self):
-    # This test can be removed if USE_WEBGPU is later included in INCLUDE_FULL_LIBRARY.
+    if config.FROZEN_CACHE:
+      # TODO(crbug.com/446944885): Make Emdawnwebgpu work with FROZEN_CACHE if possible.
+      self.skipTest("test doesn't work with frozen cache")
+    self.set_setting('DEFAULT_TO_CXX')  # emdawnwebgpu uses C++ internally
     self.build('hello_world.c', cflags=[
       '--closure=1',
       '-Werror=closure',
-      '-Wno-error=deprecated',
       '-sINCLUDE_FULL_LIBRARY',
-      '-sUSE_WEBGPU',
+      '--use-port=emdawnwebgpu',
```

Comment:
Do we have any remaining tests for `-sUSE_WEBGPU`?

Reply:
No, but I'm planning to remove it immediately after this (#25398).